readthedocs/projects/urls/private.py

```diff
@@ -337,3 +342,33 @@
 ]
 
 urlpatterns += automation_rule_urls
+
+webhook_urls = [
```

Comment:
I'm not sure if I didn't see them or there are no tests for these URLs. I would be good to add some if you haven't already.